test/topology_experimental_raft/test_tablets.py

```diff
@@ -1497,3 +1502,61 @@ async def assert_empty_table():
 
     logger.info("Verify data is not resurrected")
     await assert_empty_table()
+
+@pytest.mark.repair
+@pytest.mark.asyncio
+async def test_tablet_manual_repair(manager: ManagerClient):
```

Comment:
We should also test failure scenarios:
  1) node is lost during repair, check that removenode succeeds
  2) retry on transient failure (that table is eventually repaired).

That data is actually repaired (not just metadata updated).

Reply:
There was a such test but it was removed from this PR, Benny wanted the del api to be issued from task manager API.  The task manager integration will be on top of this PR.

Reply:
The tests I asked for are not about del API, but the repair submission API.

Reply:
The test I dropped from this PR injects errors to simulate transient failure in repair. It tests the repair will recover automatically after the transient failure is gone. I will the move the part without del api into this PR.